projects/addon-doc/components/code/index.less

```diff
@@ -19,10 +19,23 @@
     position: relative;
     margin: 0;
     white-space: normal;
-    outline: 1px solid var(--tui-border-normal);
+    border: 1px solid var(--tui-border-normal);
```

Comment:
Let's roll that line back, I think I already addressed disappeared border in my PR. With this approach you get the line under the tabs:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/519be227-b2d6-4176-8413-128da4e44244">